test/jdk/javax/swing/JColorChooser/Test8152419.java

```diff
@@ -43,9 +44,18 @@
 import javax.swing.UIManager;
 import javax.swing.border.EmptyBorder;
 
+import jtreg.SkippedException;
+
 public class Test8152419 {
 
     public static void main(String args[]) throws Exception {
+
+        // ColorChooser UI design is different for GTK L&F.
+        // There are no tabs available for GTK L&F, skip the testing.
```

Comment:
```suggestion
    public static void main(String args[]) throws Exception {
        // ColorChooser UI design is different for GTK L&F.
        // There are no tabs in GTK L&F, skip the testing.
```
No blank line at the start of a method; subtle change in comment.